frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -131,6 +131,9 @@ pub mod pallet {
     #[pallet::getter(fn force_era)]
     pub type ForceEra<T> = StorageValue<_, Forcing, ValueQuery, ForceEraOnEmpty>;
 
+    #[pallet::storage]
+    pub type PalletDisabled<T> = StorageValue<_, ()>;
```

Comment:
Noob question, but is there a reason that you use a pair of some and none instead of a boolean value?

Reply:
Also, how will this look on the client side?